src/lib/rest/OrionError.cpp

```diff
@@ -161,8 +161,8 @@ std::string OrionError::render(ConnectionInfo* ciP, const std::string& _indent)
 
   if (format == NOFORMAT)
   {
-    // Default format is XML for "v1"
-    format = XML;
+    // Default format is JSON
+    format = JSON;
   }
 
```

Comment:
I understand this entire **if** will be removed finally ...
'format' is no longer interesting as a variable. It is always JSON.

Wait, perhaps I'm too quick here ... We support common text also in some operations ...